### ISSUE TITLE
Scrolling on mobile

### ISSUE THREAD

**Issue by IVData** [claviska/jquery-minicolors#281]
Hi there.

(I've edited this because I fixed my main issue).
Im using Bootstrap 4.3.1 and JQuery 3.4.1. 

On mobile when dragging your finger around on the color picker, the page scrolls up and down (this occurs on your demo site too).

I'm calling minicolors like so:

```
<script type="text/javascript" src="js/jquery.min.js"></script>
<script type="text/javascript" src="js/bootstrap.bundle.min.js"></script>
<script type="text/javascript" src="js/jquery.minicolors.min.js"></script>
<link rel="stylesheet" href="css/bootstrap.min.css" >
<link rel="stylesheet" href="css/jquery.minicolors.css" >

<input type="text" class="form-control color-input"></input> <!-- there are a few of these -->
```

```
$('.color-input').each(function () {
    $(this).minicolors();
});
```

Any ideas?

**Comment by claviska** [comment]
Interesting. I don't recall this behavior, and I'm sure somebody would've pointed it out by now. I wonder if something changed in mobile Safari that caused this. (I'm assuming you're on iOS but correct me if I'm wrong — if not, it's an issue there, too.)

**Comment by IVData** [comment]
I'm on android =) I've tried on the built-in android browser (android 9) and up-to-date Firefox android (can't find the version) 

On 12 May 2020 8:36:46 AM NZST, Cory LaViska <notifications@github.com> wrote:
>Interesting. I don't recall this behavior, and I'm sure somebody
>would've pointed it out by now. I wonder if something changed in mobile
>Safari that caused this. (I'm assuming you're on iOS but correct me if
>I'm wrong — if not, it's an issue there, too.)>
>>
>-- >
>You are receiving this because you authored the thread.>
>Reply to this email directly or view it on GitHub:>
>https://github.com/claviska/jquery-minicolors/issues/281#issuecomment-626947750

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by IVData** [comment]
The solution to this was actually pretty easy.

After reading up on [this page](https://stackoverflow.com/questions/16348031/disable-scrolling-when-touch-moving-certain-element/43275544#43275544) it seems browsers stopped respecting the old way of doing this and the most recent answer was adding this:

```
.minicolors-panel {
  touch-action: none;
}
```

I've tested this on Chrome Android, Firefox Android, and my built-in android browser and they all work as expected (except the keyboard opens when I tap on the swatch. My next task is to stop that).

**Comment by claviska** [comment]
Nice! Let me know if you're interested in submitting these as a PR.

**Comment by claviska** [comment]
Fixed in #298.
